pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,60 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  clangStdenv
+, fetchurl
+, lib
+, # Dependencies
+  llvmPackages
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+in
+clangStdenv.mkDerivation {
+  inherit version;
```

Comment:
Couldn't this simply be moved inside the `mkDerivation` block as `version = "7.1.0";`?

Reply:
then we would need `rec` which should not be used:

https://github.com/NixOS/nixpkgs/pull/298048#discussion_r1535636936

Reply:
Alright :+1: